api/room_resource/schema.py

```diff
@@ -79,6 +79,31 @@ def mutate(self, info, **kwargs):
         return AssignResource(room_resource=room_resources)
 
 
+class DeleteAssignedResource(graphene.Mutation):
+    """
+        Delete assigned resource in a room
+    """
+    class Arguments:
+        resource_id = graphene.Int(required=True)
+        room_id = graphene.Int(required=True)
+    room_resource = graphene.Field(RoomResource)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, resource_id, room_id):
+        query = RoomResource.get_query(info)
+        room_resource = ResourceModel.query.filter_by(
+            id=resource_id).first()
+        assigned_resource = query.filter(
+            RoomResourceModel.room_id == room_id,
+            RoomResourceModel.resource_id == resource_id).first()
```

Comment:
Nice work @jesseinit. Similar to @verenceLola comment I feel there should be 2 checks. One to check if the room exists and another to check if the resource exists in the room. That way they would have more descriptive error messages

Reply:
Nice catch guys.
I felt that the check is redundant as if there is no room with such `room_id` then the resource can't be assigned in the first place.

Reply:
I think it we could be better if you raised an error `Room does not exist` before checking for the resource as its more clear if the id is not found. @jesseinit

Reply:
@kwanj-k I specifically asked for if this check was necessary when I was clarifying the scope of the task with the TC. His argument was that the record wouldn't exist if the room with the specified id doesn't exist in the first place and that was the reason I didn't perform that check.